.storybook/preview.js

```diff
@@ -13,10 +13,12 @@ import {
   DocFooter,
   MultipleStoryElementsWrapper
 } from "../src/storybook/components";
+import LinkComponent from "../src/storybook/components/link-component/link-component";
 
 addParameters({
   docs: {
     ...parameters.docs,
+    inlineStories: true,
```

Comment:
this for stop put the stories in iframe right?

Reply:
should :)